File: openapi-generator/kotlin-spring/src/main/kotlin/org/openapitools/model/DvirBaseTrailerDefects.kt
package org.openapitools.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import javax.validation.Valid
import javax.validation.constraints.*

/**
 * 
 * @param comment The comment describing the type of DVIR defect
 * @param defectType The type of DVIR defect
 */
data class DvirBaseTrailerDefects (

        @JsonProperty("comment") val comment: String? = null,

        @JsonProperty("defectType") val defectType: String? = null
) {

}

File: openapi-generator/scala-gatling/src/gatling/scala/org/openapitools/client/model/DvirBaseTrailerDefects.scala

package org.openapitools.client.model


case class DvirBaseTrailerDefects (
    /* The comment describing the type of DVIR defect */
    _comment: Option[String],
    /* The type of DVIR defect */
    _defectType: Option[String]
)
object DvirBaseTrailerDefects {
    def toStringBody(var_comment: Object, var_defectType: Object) =
        s"""
        | {
        | "comment":$var_comment,"defectType":$var_defectType
        | }
        """.stripMargin
}
